packages/contracts-bridge/contracts/BridgeRouter.sol

```diff
@@ -155,6 +158,39 @@ contract BridgeRouter is Version0, Router {
         emit Send(_token, msg.sender, _destination, _recipient, _amount, false);
     }
 
+    /**
+     * @notice Send tokens to a hook on the remote chain
+     * @param _token The token address
+     * @param _amount The token amount
+     * @param _destination The destination domain
+     * @param _hook The hook contract on the remote chain
+     * @param _extraData Extra data that will be passed to the hook for
+     *        execution
+     */
+    function xsend(
+        address _token,
+        uint256 _amount,
+        uint32 _destination,
+        bytes32 _hook,
+        bytes calldata _extraData
```

Comment:
if `_extraData` is empty, does everything still function as normal?

Reply:
cc @odyslam